unittest.cfg

```diff
@@ -0,0 +1,9 @@
+[unittest]
+verbose = 2
+plugins = ubuntu_image.testing.nose
+
+[log-capture]
+always-on = False
+
+[ubuntu-image]
+always-on = True
```

Comment:
The first section enables the nose2 plugin and turns unittest output verbose=2 by default.  I usually find that more helpful for debugging failures, but YMMV.

The second section disables the log-capture nose2 plugin.  I hate that one because it means setting pdb breakpoints are less convenient.

The third section enables our custom nose2 plugin, which provides the `-P <pattern>` convenience.  Might not seem like much now, but when we have a big test suite, it's soooo much nicer for narrowing down to debugging failures.